solr/core/src/java/org/apache/solr/handler/component/SearchHandler.java

```diff
@@ -502,6 +502,7 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
               sreq.actualShards = rb.shards;
             }
             sreq.responses = new ArrayList<>(sreq.actualShards.length); // presume we'll get a response from each shard we send to
+            sreq.requestStartTime = sreq.requestStartTime;
```

Comment:
Should we set to current time instead? :)

Reply:
No, it may give us some more hint to debug